sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -132,24 +190,7 @@ export interface PeekMessagesOptions extends OperationOptionsBase {
 }
 
 // @public
-export interface QueueProperties {
-    authorizationRules?: AuthorizationRule[];
-    autoDeleteOnIdle?: string;
-    deadLetteringOnMessageExpiration?: boolean;
-    defaultMessageTtl?: string;
-    duplicateDetectionHistoryTimeWindow?: string;
-    enableBatchedOperations?: boolean;
-    enablePartitioning?: boolean;
-    forwardDeadLetteredMessagesTo?: string;
-    forwardTo?: string;
-    lockDuration?: string;
-    maxDeliveryCount?: number;
-    maxSizeInMegabytes?: number;
-    name: string;
-    requiresDuplicateDetection?: boolean;
-    requiresSession?: boolean;
-    status?: EntityStatus;
-    userMetadata?: string;
+export interface QueueProperties extends Required<Pick<CreateQueueOptions, UpdatableFieldsForQueue>>, Readonly<Pick<CreateQueueOptions, AllowUndefinedFieldsForQueue>>, Readonly<Required<Omit<CreateQueueOptions, UpdatableFieldsForQueue | AllowUndefinedFieldsForQueue>>> {
```

Comment:
Am curious to see how this shows up in our docs. Do we have a prior example where we have used `Required`, `Pick` or `Omit` in public types?

Reply:
Here is an example from App Config:

Link: https://docs.microsoft.com/en-us/javascript/api/@azure/app-configuration/configurationsettingresponse?view=azure-node-latest

Picture:

<img width="934" alt="Screen Shot 2020-07-21 at 5 48 02 PM" src="https://user-images.githubusercontent.com/16890566/88121354-57efbf80-cb7a-11ea-9b4e-7163ad7611b0.png">

Reply:
![image](https://user-images.githubusercontent.com/10452642/88153978-0582c300-cbbb-11ea-933e-9bee7e28ec45.png)
Docs are bad, going with duplicating the properties in CreateEntityOptions and EntityProperties.